auditlog/models/log.py

```diff
@@ -9,7 +10,11 @@ class AuditlogLog(models.Model):
     _order = "create_date desc"
 
     name = fields.Char("Resource Name", size=64)
-    model_id = fields.Many2one("ir.model", string="Model")
+    model_id = fields.Many2one(
+        "ir.model", string="Model", index=True, ondelete="set null"
+    )
+    model_name = fields.Char(readonly=True)
+    model_model = fields.Char(string="Technical Model Name", readonly=True)
```

Comment:
Stupid question probably, but why aren't these stored related fields?

Reply:
Pieter, you *wrote* this code ;-) The reason they are not stored related fields is because they will get wiped when the field is deleted, whereas the exact point of these fields is to keep the info when the field is deleted.

Reply:
This OK for you @PieterPaulussen?